transformations/sentence_additions/README.md

```diff
@@ -0,0 +1,41 @@
+# Sentence Additions 🦎  + ⌨️ → 🐍
+This perturbation adds generated sentence to all types of text sources (sentence, paragraph, etc.) by passing the input text to a GPT-2 Text Generation model.
+
+Author name: Timothy Sum Hon Mun (timothy22000@gmail.com)
+
+## What type of a transformation is this?
+This transformation will take in an input text and generate additional sentences to perturbe the inputs that have been shown by Jia and Liang, 2017 to impact model performance. This can be used to test model robustness. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks on text classification and generation.
+
+## Related work
+
```

Comment:
Please also take a look at this PR: https://github.com/GEM-benchmark/NL-Augmenter/pull/131/files

Reply:
Hi Kaustubh,

Great to hear from you and I hope you are doing well!

I have taken a look at that PR and can see that it also has been inspired by the same paper but for the AddAny method whereas I was inspired by the AddSent and AddOneSent method in the paper. 

The AddAny method adds a sequence of n arbitrary words to perturbed the input text (eg "offered consistent operational e.g. observations precise conceptual intuitive." from their test case). On the other hand, the AddSent method adds proper grammatical sentences to the input text instead (eg "The man travelled to the past to change history."). 

The AddSent/AddOneSent has also been shown to be able to fool different models with the same generated adversarial example much better in comparison to the AddAny method so the perturbatation has better transferability across models.

Both the PR and the methods in the paper are applied to the Q&A setting. In particular, the AddSent method requires mutation of the original question asked and generating a fake answer based on that. Then, converting them into a statement that serves as the answer as an adversarial example.

I wanted a transformation that can introduce grammatical sentences without needing a question to be mutated and a fake answer to be generated so it can be applied to any text. Therefore, I propose this transformation that is independent of any question asked.